### PR TITLE
chore: Create initial setup for Java app

### PR DESCRIPTION
**Pull request by kkafar**
Add initial setup for Java application with JavaFX -- we can change it later on, if we decide to go with different technology

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        exclude(group: 'org.openjfx')
+    }
+//    implementation('net.synedra:validatorfx:0.1.13') {
+//        exclude(group: 'org.openjfx')
+//    }
+    implementation('org.kordamp.ikonli:ikonli-javafx:12.3.0')
+    implementation('org.kordamp.bootstrapfx:bootstrapfx-core:0.4.0')
+//    implementation('eu.hansolo:tilesfx:17.1.5') {
+//        exclude(group: 'org.openjfx')
+//    }
```
Comment (kkafar) [inline build.gradle:48]: The project did not build successfully with these two dependencies. Had to cut them out. Don't think we will need them. Even if so, we can figure it out later on.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```
Comment (co012) [inline gradle/wrapper/gradle-wrapper.properties:3]: Let's bump the version to 7.4.1 (Maybe)

**Review by co012 — COMMENTED** [review]
Why did you use javafx version 15.0.1 (There are newer versions).

**Comment by kkafar** [comment]
I upgraded gradle to 7.4.1 [(latest up to date)](https://github.com/gradle/gradle/releases) & JavaFX to 17.0.2 [(latest LTS version)](https://gluonhq.com/products/javafx/openjfx-17-release-notes/).
I wonder if there were any particular fixes / features introduced we could be interested in, @co012? :D

**Review by mhawryluk — APPROVED** [review]
I managed to run the app on my computer, so seems good!

**Pull request merged**